apiHelpers/atelier.js

```diff
@@ -1 +1,61 @@
-const endpoint = 'https://app-hrsei-api.herokuapp.com/api/fec2/:hr-la/'
\ No newline at end of file
+const endpoint = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-lax'; //note that all endpoints must contain a '/' after 'hr-lax'
+const credentials = require('../config.js')
+// import credentials from '../config.js'
+const axios = require ('axios');
+
+module.exports = {
+  getProducts: function(callback) {
```

Comment:
Due to using axios and promise syntax, we don't need a callback here. This is personal, but we can have a few options:

1. the getProducts takes in the (req, res) objects. This does not need to be explicitly passed from the router as express handles this. After doing the axios call, the getProducts function can directly send data back through res.status(). If doing this method, the router simply needs to call API.getproducts as a callback function and nothing else.
2. We simply return the axios results. This can look be as simple as 'return axios([ config data here])', OR you can do a 'return res' in the .then block. This will simply pass on the data to the router, and the router will do the error handling on its end, without the need for the error first callback pattern

Reply:
One thing to add to above - To start a promise chain from the atelier.js side, you HAVE to return the axios call itself (just tested)

Reply:
Kindof like the second option for reading with the 'return axios([ config data here])' and the first option in your comment with product.js